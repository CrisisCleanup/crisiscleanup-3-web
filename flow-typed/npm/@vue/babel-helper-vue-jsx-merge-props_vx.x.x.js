// flow-typed signature: d7a23c53f4091ef2bf1df8afa5eb43c4
// flow-typed version: <<STUB>>/@vue/babel-helper-vue-jsx-merge-props_v^1.0.0/flow_v0.131.0

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@vue/babel-helper-vue-jsx-merge-props'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@vue/babel-helper-vue-jsx-merge-props' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */
declare module '@vue/babel-helper-vue-jsx-merge-props/dist/helper' {
  declare module.exports: any;
}

// Filename aliases
declare module '@vue/babel-helper-vue-jsx-merge-props/dist/helper.js' {
  declare module.exports: $Exports<'@vue/babel-helper-vue-jsx-merge-props/dist/helper'>;
}
