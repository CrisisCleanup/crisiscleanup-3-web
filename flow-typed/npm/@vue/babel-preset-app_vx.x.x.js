// flow-typed signature: 6ba91a92845d9a556a5bcdcc4f033fed
// flow-typed version: <<STUB>>/@vue/babel-preset-app_v^4.3.1/flow_v0.129.0

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@vue/babel-preset-app'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@vue/babel-preset-app' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */
declare module '@vue/babel-preset-app/polyfillsPlugin' {
  declare module.exports: any;
}

// Filename aliases
declare module '@vue/babel-preset-app/index' {
  declare module.exports: $Exports<'@vue/babel-preset-app'>;
}
declare module '@vue/babel-preset-app/index.js' {
  declare module.exports: $Exports<'@vue/babel-preset-app'>;
}
declare module '@vue/babel-preset-app/polyfillsPlugin.js' {
  declare module.exports: $Exports<'@vue/babel-preset-app/polyfillsPlugin'>;
}
