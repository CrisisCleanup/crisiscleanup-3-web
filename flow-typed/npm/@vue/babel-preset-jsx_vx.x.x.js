// flow-typed signature: eeecaeff29e4a3a6c63c4fe7e16a0eb3
// flow-typed version: <<STUB>>/@vue/babel-preset-jsx_v^1.1.2/flow_v0.131.0

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@vue/babel-preset-jsx'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@vue/babel-preset-jsx' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */
declare module '@vue/babel-preset-jsx/dist/plugin.cjs' {
  declare module.exports: any;
}

// Filename aliases
declare module '@vue/babel-preset-jsx/dist/plugin.cjs.js' {
  declare module.exports: $Exports<'@vue/babel-preset-jsx/dist/plugin.cjs'>;
}
