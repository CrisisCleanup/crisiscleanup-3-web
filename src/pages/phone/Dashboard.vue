<template>
  <Loader :loading="loading" class="h-full w-full">
    <template #content>
      <phone-layout>
        <template #grid-start>
          <div class="grid--start">
            <agent-block />
          </div>
        </template>
        <template #grid-main>
          <div class="grid--main w-auto">
            <div class="dash-grid">
              <div class="grid--leader">
                <leaderboard />
              </div>
              <div class="grid--train">
                <training-card
                  :image-path="training.imagePath"
                  :description="training.description"
                  :time-to-complete="training.timeToComplete"
                >
                </training-card>
                <Table :columns="contactsCols" :data="contactMetrics" />
              </div>
            </div>
          </div>
        </template>
      </phone-layout>
    </template>
  </Loader>
</template>

<script>
import User from '@/models/User';
import { mapActions, mapGetters } from 'vuex';
import Loader from '@/components/Loader.vue';
import NewsTrainingCard from '@/components/phone/NewsTrainingCard.vue';
import Leaderboard from '@/components/phone/Leaderboard.vue';
import PhoneLayout from '@/layouts/Phone.vue';
import AgentBlock from '@/components/phone/blocks/Agent.vue';
import Table from '@/components/Table.vue';

export default {
  name: 'Phone',
  components: {
    PhoneLayout,
    AgentBlock,
    Loader,
    Table,

    'training-card': NewsTrainingCard,
    Leaderboard,
  },
  data() {
    return {
      loading: false,
      training: {
        imagePath: require('@/assets/newstrainingss.jpg'),
        description:
          'Then go and tempor incididunt ut labore et dolore magna aliqua.',
        timeToComplete: '10 minutes',
      },
      contactsCols: [
        {
          dataIndex: 'contact_id',
          key: 'contact_id',
          title: 'Contact',
          width: '1fr',
        },
        {
          dataIndex: 'state',
          key: 'state',
          title: 'State',
          width: '1fr',
        },
        {
          dataIndex: 'action',
          key: 'action',
          title: 'Action',
          width: '1fr',
        },
        {
          dataIndex: 'entered_timestamp',
          key: 'entered_timestamp',
          title: 'Timestamp',
          width: '1fr',
        },
        {
          dataIndex: 'agent_id',
          key: 'agent_id',
          title: 'Agent',
          width: '1fr',
        },
      ],
    };
  },
  computed: {
    currentUser() {
      return User.find(this.$store.getters['auth/userId']);
    },
    ...mapGetters('phone', [
      'connectReady',
      'agentState',
      'contactState',
      'callIncoming',
      'contactMetrics',
    ]),
  },

  methods: {
    ...mapActions('phone', ['getRealtimeMetrics']),
  },
  async mounted() {
    this.loading = true;
    await this.getRealtimeMetrics();
    this.loading = false;
  },
};
</script>

<style scoped lang="scss">
$areas: leader train story;

.grid--main {
  .dash-grid {
    display: inline-grid;
    grid-gap: 2rem;
    grid:
      [r1] 'leader' [r1end]
      [r2] 'train' [r2end]
      [r3] 'story' [r3end]
      / auto;
    .grid {
      @each $area in $areas {
        &--#{$area} {
          grid-area: $area;
        }
      }
    }
  }
}
</style>
