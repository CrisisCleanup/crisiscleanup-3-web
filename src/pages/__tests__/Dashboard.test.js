/**
 *
 * Tests for Dashboard
 *
 * Components
 */

import { createLocalVue, shallowMount } from '@vue/test-utils';
import Vuex from 'vuex';
import { Store } from 'vuex-mock-store';
import Table from '@/components/Table';
import Dashboard from '../Dashboard';

jest.mock('@/models/User');
jest.mock('@/models/Status');
jest.mock('@/models/Worksite');
jest.mock('@/models/Incident');
jest.mock('@/models/Organization');
jest.mock('@/models/WorksiteRequest');

const localVue = createLocalVue();
localVue.use(Vuex);
localVue.filter('numeral', (n) => n);
localVue.filter('getColorForWorkType', () => '#ffffff');
localVue.directive('can', () => false);

const $route = {
  path: '/incident/99/dashboard',
  params: {
    incident_id: 99,
  },
};

const store = new Store({
  state: {
    incident: {
      currentIncidentId: 99,
    },
    auth: {
      user: {
        id: 1,
        first_name: 'First',
        last_name: 'Last',
        organization: {
          id: 1,
          name: 'Test Organization',
        },
      },
    },
  },
});

const mocks = {
  $route,
  $store: store,
  $can: () => true,
  $http: {
    get: () => {
      return {
        data: {
          count: 2,
          results: [],
        },
      };
    },
  },
  $t: (key) => key,
};

const mountWithOptions = () =>
  shallowMount(Dashboard, {
    stubs: {
      'base-button': true,
      'base-input': true,
      'ccu-icon': true,
      badge: true,
      spinner: true,
      'router-link': true,
      table: Table,
    },
    mocks,
    localVue,
  });

describe('Dashboard', () => {
  afterEach(() => store.reset());
  it('should not log any errors', () => {
    const spy = jest.spyOn(global.console, 'error');
    mountWithOptions();
    expect(spy).not.toHaveBeenCalled();
  });

  it('should render correctly and match snapshot', () => {
    const wrapper = mountWithOptions();
    expect(wrapper.element).toMatchSnapshot();
  });
});
