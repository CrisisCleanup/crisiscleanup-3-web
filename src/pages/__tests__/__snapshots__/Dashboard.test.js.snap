// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard should render correctly and match snapshot 1`] = `
<loader-stub
  class="p-6 bg-crisiscleanup-light-grey h-full overflow-auto"
>
  <div
    class="flex flex-col"
  >
    <div
      class="flex self-end mr-4"
    >
      <redeployrequest-stub />
       
      <inviteusers-stub
        class="mx-1"
      />
    </div>
     
    <div
      class="flex flex-wrap"
    >
      <div
        class="w-full md:w-1/2 xl:w-1/4 p-3 cursor-pointer"
      >
        <div
          class="bg-crisiscleanup-lightblue-100 rounded-lg shadow-lg p-5"
        >
          <div
            class="flex flex-row items-center"
          >
            <div
              class="flex-shrink pr-4"
            >
              <div
                class="rounded-full p-1 bg-crisiscleanup-lightblue-900"
              >
                <img
                  src="@/assets/icons/cases.svg"
                  style="height: 50px;"
                />
              </div>
            </div>
             
            <div
              class="flex-1 text-right md:text-center"
            >
              <h5
                class="uppercase text-grey-900"
              >
                
                  dashboard.my_claimed_cases (Texas Flooding, Oct 2018)
                
              </h5>
               
              <h3
                class="text-3xl"
              >
                
                  
                  
                <span
                  class="text-green"
                >
                  <i
                    class="fas fa-caret-up"
                  />
                </span>
              </h3>
            </div>
          </div>
        </div>
      </div>
       
      <div
        class="w-full md:w-1/2 xl:w-1/4 p-3 cursor-pointer"
      >
        <div
          class="bg-crisiscleanup-yellow-100 rounded-lg shadow-lg"
        >
          <div
            class="flex flex-row items-center p-5"
          >
            <div
              class="flex-shrink pr-4"
            >
              <div
                class="rounded-full p-1 bg-crisiscleanup-yellow-900"
              >
                <img
                  src="@/assets/icons/cases.svg"
                  style="height: 50px;"
                />
              </div>
            </div>
             
            <div
              class="flex-1 text-right md:text-center"
            >
              <h5
                class="uppercase text-grey-900"
              >
                
                  dashboard.total_claimed (Texas Flooding, Oct 2018)
                
              </h5>
               
              <h3
                class="text-3xl"
              >
                
                  0
                  
                <span
                  class="text-base"
                >
                  
                    (NaN
                    dashboard.of_total)
                  
                </span>
                 
                <span
                  class="text-orange"
                >
                  <i
                    class="fas fa-exchange-alt"
                  />
                </span>
              </h3>
            </div>
          </div>
           
          <div
            class="
                bottom-0
                left-0
                border-b-4
                rounded-b-lg
                border-yellow-600
              "
          />
        </div>
      </div>
       
      <div
        class="w-full md:w-1/2 xl:w-1/4 p-3 cursor-pointer"
      >
        <div
          class="bg-orange-400 rounded-lg shadow-lg"
        >
          <div
            class="flex flex-row items-center p-5"
          >
            <div
              class="flex-shrink pr-4"
            >
              <div
                class="rounded-full p-1 bg-orange-600"
              >
                <img
                  src="@/assets/icons/cases.svg"
                  style="height: 50px;"
                />
              </div>
            </div>
             
            <div
              class="flex-1 text-right md:text-center"
            >
              <h5
                class="uppercase text-grey-900"
              >
                
                  dashboard.in_progress (Texas Flooding, Oct 2018)
                
              </h5>
               
              <h3
                class="text-3xl"
              >
                
                  0
                  
                <span
                  class="text-base"
                >
                  
                    (NaN
                    dashboard.of_total)
                  
                </span>
                 
                <span
                  class="text-yellow-900"
                >
                  <i
                    class="fas fa-caret-up"
                  />
                </span>
              </h3>
            </div>
          </div>
           
          <div
            class="
                bottom-0
                left-0
                border-b-4
                rounded-b-lg
                border-orange-600
              "
          />
        </div>
      </div>
       
      <div
        class="w-full md:w-1/2 xl:w-1/4 p-3 cursor-pointer"
      >
        <div
          class="bg-crisiscleanup-green-100 rounded-lg shadow-lg"
        >
          <div
            class="flex flex-row items-center p-5"
          >
            <div
              class="flex-shrink pr-4"
            >
              <div
                class="rounded-full p-1 bg-crisiscleanup-green-900"
              >
                <img
                  src="@/assets/icons/cases.svg"
                  style="height: 50px;"
                />
              </div>
            </div>
             
            <div
              class="flex-1 text-right md:text-center"
            >
              <h5
                class="uppercase text-grey-900"
              >
                
                  dashboard.closed (Texas Flooding, Oct 2018)
                
              </h5>
               
              <h3
                class="text-3xl"
              >
                
                  0
                  
                <span
                  class="text-base"
                >
                  
                    (NaN
                    dashboard.of_total)
                  
                </span>
              </h3>
            </div>
          </div>
           
          <div
            class="bottom-0 left-0 border-b-4 rounded-b-lg border-green-600"
          />
        </div>
      </div>
    </div>
     
    <div
      class="flex"
    >
      <div
        class="m-4 pt-2 shadow bg-white w-full"
      >
        <div
          class="py-4 px-4 text-gray-500 border-b"
        >
          
            dashboard.my_cases
          
        </div>
         
        <div
          class="p-4"
        >
          <div
            class="table-grid w-full"
          >
            <div
              class="header text-crisiscleanup-grey-700 bg-white"
              style="display: grid; grid-template-columns: 0.75fr 2fr 1fr 1fr 1fr 1fr;"
            >
              <!---->
               
              <!---->
               
              <div
                class="p-2 border-b flex items-center cursor-pointer header-column"
              >
                <base-text-stub
                  class="text-crisiscleanup-grey-700"
                  regular=""
                  variant="h3"
                >
                  
          dashboard.no
        
                </base-text-stub>
                 
                <div>
                  <ccu-icon-stub
                    alt="actions.sortable"
                    size="small"
                    type="updown"
                  />
                </div>
              </div>
              <div
                class="p-2 border-b flex items-center cursor-pointer header-column"
              >
                <base-text-stub
                  class="text-crisiscleanup-grey-700"
                  regular=""
                  variant="h3"
                >
                  
          dashboard.work_type
        
                </base-text-stub>
                 
                <!---->
              </div>
              <div
                class="p-2 border-b flex items-center cursor-pointer header-column"
              >
                <base-text-stub
                  class="text-crisiscleanup-grey-700"
                  regular=""
                  variant="h3"
                >
                  
          dashboard.name
        
                </base-text-stub>
                 
                <div>
                  <ccu-icon-stub
                    alt="actions.sortable"
                    size="small"
                    type="updown"
                  />
                </div>
              </div>
              <div
                class="p-2 border-b flex items-center cursor-pointer header-column"
              >
                <base-text-stub
                  class="text-crisiscleanup-grey-700"
                  regular=""
                  variant="h3"
                >
                  
          dashboard.full_address
        
                </base-text-stub>
                 
                <!---->
              </div>
              <div
                class="p-2 border-b flex items-center cursor-pointer header-column"
              >
                <base-text-stub
                  class="text-crisiscleanup-grey-700"
                  regular=""
                  variant="h3"
                >
                  
          dashboard.phone
        
                </base-text-stub>
                 
                <!---->
              </div>
              <div
                class="p-2 border-b flex items-center cursor-pointer header-column"
              >
                <base-text-stub
                  class="text-crisiscleanup-grey-700"
                  regular=""
                  variant="h3"
                >
                  
          
        
                </base-text-stub>
                 
                <!---->
              </div>
               
              <div
                class="
          p-2
          border-b
          flex
          items-center
          cursor-pointer
          bg-crisiscleanup-light-grey
        "
              >
                <base-input-stub
                  input-style="width: 100%"
                  placeholder="dashboard.no"
                />
              </div>
              <div
                class="
          p-2
          border-b
          flex
          items-center
          cursor-pointer
          bg-crisiscleanup-light-grey
        "
              >
                <form-select-stub
                  class="w-64 bg-white border h-10 border-crisiscleanup-dark-100"
                  item-key="value"
                  label="name_t"
                  options=""
                />
              </div>
              <div
                class="
          p-2
          border-b
          flex
          items-center
          cursor-pointer
          bg-crisiscleanup-light-grey
        "
              >
                <base-input-stub
                  input-style="width: 100%"
                  placeholder="dashboard.name"
                />
              </div>
              <div
                class="
          p-2
          border-b
          flex
          items-center
          cursor-pointer
          bg-crisiscleanup-light-grey
        "
              >
                <base-input-stub
                  input-style="width: 100%"
                  placeholder="dashboard.full_address"
                />
              </div>
              <div
                class="
          p-2
          border-b
          flex
          items-center
          cursor-pointer
          bg-crisiscleanup-light-grey
        "
              >
                <base-input-stub
                  input-style="width: 100%"
                  placeholder="dashboard.phone"
                />
              </div>
              <div
                class="
          p-2
          border-b
          flex
          items-center
          cursor-pointer
          bg-crisiscleanup-light-grey
        "
              >
                <!---->
              </div>
            </div>
             
            <div
              class="body bg-white relative"
              style="max-height: 600px; overflow: auto; height: 300px;"
            >
              <!---->
               
              <div
                class="hover:bg-crisiscleanup-light-grey"
                style="display: grid; grid-template-columns: 0.75fr 2fr 1fr 1fr 1fr 1fr;"
              >
                <!---->
                 
                <!---->
                 
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <div
                    class="flex flex-wrap w-full"
                  />
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <div
                    class="flex"
                  >
                    <router-link-stub
                      tag="div"
                      to="/incident/99/cases/undefined/edit?showOnMap=true"
                    >
                      <ccu-icon-stub
                        alt="actions.jump_to_case"
                        class="p-1 py-2 w-8"
                        linked="true"
                        size="medium"
                        type="go-case"
                      />
                    </router-link-stub>
                     
                    <ccu-icon-stub
                      alt="actions.print"
                      class="p-1 py-2 w-8"
                      size="medium"
                      type="print"
                    />
                  </div>
                </div>
                 
                <!---->
              </div>
              <div
                class="hover:bg-crisiscleanup-light-grey"
                style="display: grid; grid-template-columns: 0.75fr 2fr 1fr 1fr 1fr 1fr;"
              >
                <!---->
                 
                <!---->
                 
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <div
                    class="flex flex-wrap w-full"
                  />
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <div
                    class="flex"
                  >
                    <router-link-stub
                      tag="div"
                      to="/incident/99/cases/undefined/edit?showOnMap=true"
                    >
                      <ccu-icon-stub
                        alt="actions.jump_to_case"
                        class="p-1 py-2 w-8"
                        linked="true"
                        size="medium"
                        type="go-case"
                      />
                    </router-link-stub>
                     
                    <ccu-icon-stub
                      alt="actions.print"
                      class="p-1 py-2 w-8"
                      size="medium"
                      type="print"
                    />
                  </div>
                </div>
                 
                <!---->
              </div>
              <div
                class="hover:bg-crisiscleanup-light-grey"
                style="display: grid; grid-template-columns: 0.75fr 2fr 1fr 1fr 1fr 1fr;"
              >
                <!---->
                 
                <!---->
                 
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <div
                    class="flex flex-wrap w-full"
                  />
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <div
                    class="flex"
                  >
                    <router-link-stub
                      tag="div"
                      to="/incident/99/cases/undefined/edit?showOnMap=true"
                    >
                      <ccu-icon-stub
                        alt="actions.jump_to_case"
                        class="p-1 py-2 w-8"
                        linked="true"
                        size="medium"
                        type="go-case"
                      />
                    </router-link-stub>
                     
                    <ccu-icon-stub
                      alt="actions.print"
                      class="p-1 py-2 w-8"
                      size="medium"
                      type="print"
                    />
                  </div>
                </div>
                 
                <!---->
              </div>
              <div
                class="hover:bg-crisiscleanup-light-grey"
                style="display: grid; grid-template-columns: 0.75fr 2fr 1fr 1fr 1fr 1fr;"
              >
                <!---->
                 
                <!---->
                 
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <div
                    class="flex flex-wrap w-full"
                  />
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <div
                    class="flex"
                  >
                    <router-link-stub
                      tag="div"
                      to="/incident/99/cases/undefined/edit?showOnMap=true"
                    >
                      <ccu-icon-stub
                        alt="actions.jump_to_case"
                        class="p-1 py-2 w-8"
                        linked="true"
                        size="medium"
                        type="go-case"
                      />
                    </router-link-stub>
                     
                    <ccu-icon-stub
                      alt="actions.print"
                      class="p-1 py-2 w-8"
                      size="medium"
                      type="print"
                    />
                  </div>
                </div>
                 
                <!---->
              </div>
              <div
                class="hover:bg-crisiscleanup-light-grey"
                style="display: grid; grid-template-columns: 0.75fr 2fr 1fr 1fr 1fr 1fr;"
              >
                <!---->
                 
                <!---->
                 
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <div
                    class="flex flex-wrap w-full"
                  />
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <div
                    class="flex"
                  >
                    <router-link-stub
                      tag="div"
                      to="/incident/99/cases/undefined/edit?showOnMap=true"
                    >
                      <ccu-icon-stub
                        alt="actions.jump_to_case"
                        class="p-1 py-2 w-8"
                        linked="true"
                        size="medium"
                        type="go-case"
                      />
                    </router-link-stub>
                     
                    <ccu-icon-stub
                      alt="actions.print"
                      class="p-1 py-2 w-8"
                      size="medium"
                      type="print"
                    />
                  </div>
                </div>
                 
                <!---->
              </div>
              <div
                class="hover:bg-crisiscleanup-light-grey"
                style="display: grid; grid-template-columns: 0.75fr 2fr 1fr 1fr 1fr 1fr;"
              >
                <!---->
                 
                <!---->
                 
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <div
                    class="flex flex-wrap w-full"
                  />
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <!---->
                   
                  <!---->
                </div>
                <div
                  class="flex items-center p-2 lg:border-b md:border-b cursor-pointer"
                >
                  <div
                    class="flex"
                  >
                    <router-link-stub
                      tag="div"
                      to="/incident/99/cases/undefined/edit?showOnMap=true"
                    >
                      <ccu-icon-stub
                        alt="actions.jump_to_case"
                        class="p-1 py-2 w-8"
                        linked="true"
                        size="medium"
                        type="go-case"
                      />
                    </router-link-stub>
                     
                    <ccu-icon-stub
                      alt="actions.print"
                      class="p-1 py-2 w-8"
                      size="medium"
                      type="print"
                    />
                  </div>
                </div>
                 
                <!---->
              </div>
               
              <div
                class="p-2 text-crisiscleanup-grey-700 italic"
              >
                
      info.no_items_found
    
              </div>
            </div>
             
            <!---->
          </div>
        </div>
      </div>
    </div>
     
    <div
      class="flex"
    >
      <div
        class="w-full m-4 pt-2 shadow bg-white flex-shrink"
      >
        <div
          class="py-4 px-4 text-gray-500 border-b"
        >
          
            dashboard.case_transfer_requests
          
        </div>
         
        <div
          class="py-4 px-4 border-b flex items-center"
        >
          <base-button-stub
            class="mr-2 border-r pr-2 text-primary-dark"
            size="medium"
            text="dashboard.inbound_requests"
            variant="text"
          />
           
          <base-button-stub
            class="mr-2 border-r pr-2"
            size="medium"
            text="dashboard.outbound_requests"
            variant="text"
          />
           
          <base-button-stub
            class="mr-2"
            size="medium"
            text="dashboard.archived_requests"
            variant="text"
          />
        </div>
         
        <div
          class="p-4"
        >
          <div
            class="table-grid w-full"
          >
            <div
              class="header text-crisiscleanup-grey-700 bg-white"
              style="display: grid; grid-template-columns: 80px 10% 0.5fr 1fr 1fr 1fr 1fr;"
            >
              <!---->
               
              <!---->
               
              <div
                class="p-2 border-b flex items-center cursor-pointer header-column"
              >
                <base-text-stub
                  class="text-crisiscleanup-grey-700"
                  regular=""
                  variant="h3"
                >
                  
          dashboard.no
        
                </base-text-stub>
                 
                <div>
                  <ccu-icon-stub
                    alt="actions.sortable"
                    size="small"
                    type="updown"
                  />
                </div>
              </div>
              <div
                class="p-2 border-b flex items-center cursor-pointer header-column"
              >
                <base-text-stub
                  class="text-crisiscleanup-grey-700"
                  regular=""
                  variant="h3"
                >
                  
          dashboard.work_type
        
                </base-text-stub>
                 
                <!---->
              </div>
              <div
                class="p-2 border-b flex items-center cursor-pointer header-column"
              >
                <base-text-stub
                  class="text-crisiscleanup-grey-700"
                  regular=""
                  variant="h3"
                >
                  
          dashboard.status
        
                </base-text-stub>
                 
                <!---->
              </div>
              <div
                class="p-2 border-b flex items-center cursor-pointer header-column"
              >
                <base-text-stub
                  class="text-crisiscleanup-grey-700"
                  regular=""
                  variant="h3"
                >
                  
          dashboard.org_label
        
                </base-text-stub>
                 
                <!---->
              </div>
              <div
                class="p-2 border-b flex items-center cursor-pointer header-column"
              >
                <base-text-stub
                  class="text-crisiscleanup-grey-700"
                  regular=""
                  variant="h3"
                >
                  
          dashboard.last_action
        
                </base-text-stub>
                 
                <!---->
              </div>
              <div
                class="p-2 border-b flex items-center cursor-pointer header-column"
              >
                <base-text-stub
                  class="text-crisiscleanup-grey-700"
                  regular=""
                  variant="h3"
                >
                  
          dashboard.next_action
        
                </base-text-stub>
                 
                <!---->
              </div>
              <div
                class="p-2 border-b flex items-center cursor-pointer header-column"
              >
                <base-text-stub
                  class="text-crisiscleanup-grey-700"
                  regular=""
                  variant="h3"
                >
                  
          
        
                </base-text-stub>
                 
                <!---->
              </div>
               
              <!---->
            </div>
             
            <div
              class="body bg-white relative"
              style="max-height: 600px; overflow: auto; height: 300px;"
            >
              <!---->
                
              <div
                class="p-2 text-crisiscleanup-grey-700 italic"
              >
                
      info.no_items_found
    
              </div>
            </div>
             
            <!---->
          </div>
        </div>
      </div>
    </div>
     
    <div
      class="flex"
    >
      <div
        class="w-full m-4 pt-2 shadow bg-white flex-shrink"
      >
        <div
          class="py-4 px-4 text-gray-500 border-b"
        >
          
            dashboard.user_transfer_requests
          
        </div>
         
        <div
          class="p-4"
        >
          <usertransferrequesttable-stub
            requests=""
          />
        </div>
      </div>
    </div>
  </div>
</loader-stub>
`;
